### PR TITLE
feat(AAiT-web_g3): add footer page

### PR DESCRIPTION
**Pull request by dawit-andargachew**
We have added footer page. and here are the main changes that we make
 - component/footer:  Footer page is added here
 - public/assets:  Added SVG files that are used in the footer page

## for desktop
image

## for mobile screen

image

image

### CODE REVIEW

File: aait/web/group-3/starter_project_group3/components/footer/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+            <Link href={"#"}>Executives</Link>
+            <Link href={"#"}>Staffs</Link>
+          </div>
+
+          {/* Blogs */}
+          <div className=" flex justify-center flex-col font-light gap-y-5">
+            <p className=" font-semibold">Blogs</p>
+
+            <Link href={"#"}>Recent Blogs</Link>
+            <Link href={"#"}>New Blogs</Link>
```
Comment (kidus101) [inline aait/web/group-3/starter_project_group3/components/footer/Footer.tsx:66]: Remove the spacing on line 64.

**Review by kidus101 — COMMENTED** [review]
Edit the parts that I have commented on.

File: aait/web/group-3/starter_project_group3/components/footer/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+          </div>
+        </section>
+      </div>
```
[the remark this answers is not in the document]
Comment (dawit-andargachew) [reply aait/web/group-3/starter_project_group3/components/footer/Footer.tsx:69]: fixed.
footerlinks are stored in the array and used map function to map those links

File: aait/web/group-3/starter_project_group3/components/footer/Footer.tsx (hunk 1 above)
Comment (dawit-andargachew) [reply aait/web/group-3/starter_project_group3/components/footer/Footer.tsx:66]: removed

**Pull request merged**